lib/core/enums/amplitude_events.dart

```diff
@@ -115,6 +115,23 @@ enum AmplitudeEvents {
   generosityChallengeRegistrationSucceeded(
     'generosity_challenge_registration_succeeded',
   ),
+  // Registration flow
+  newUserContinuedOnEmailSignUpPage(
+    'new_user_continued_on_email_sign_up_page',
+  ),
+  tempUserContinuedOnEmailSignUpPage(
+    'temp_user_continued_on_email_sign_up_page',
+  ),
+  newUserFilledInPersonalInfo('new_user_filled_in_personal_info'),
+  newUserFilledStripeDetailsSuccessfully(
+    'new_user_filled_stripe_details_successfully',
+  ),
+  newUserClosedStripeSheetIncomplete(
+    'new_user_closed_stripe_sheet_imcomplete',
+  ),
+  clickedSetUpFamilyFromRegistrationSuccess(
```

Comment:
most (recents) events have the action ('Clicked') as last part of the name. For example:
helpMeFindCharityPressed
deeplinkCoinScanned
schoolEventFlowStartButtonClicked

I would keep that format :)

so something like:
registration_stripe_sheet_filled
xxx_incomplete_closed
xx_registration_succes_button_clicked

Reply:
changed! :)